apps/back-office/translations/nl.json

```diff
@@ -25,6 +25,8 @@
       "state": "Status",
       "geo_location": "Location"
     },
+    "title": "Melding {meldingId}",
+    "back-link": "Overzicht meldingen",
```

Comment:
We generally order these in order of appearance, so first `back-link`, then `title`, then `terms` etc.